### PR TITLE
[PT-BR] update `BanchoBot`

### PR DESCRIPTION
**Pull request by GranularBow8193**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] ~~*(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)~~ been 2 weeks

### CODE REVIEW

File: wiki/BanchoBot/pt-br.md [hunk 1]
```diff
@@ -1,40 +1,160 @@
...
+
+```
+13:00 pippi: !help
+13:00 BanchoBot: Standard Commands (!COMMAND or /msg BanchoBot COMMAND):
+13:00 BanchoBot: WHERE <usário>
+13:00 BanchoBot: STATS <usuário>
+13:00 BanchoBot: FAQ <item>|list
+13:00 BanchoBot: REPORT <motivo> - chame um administrador
+13:00 BanchoBot: REQUEST [lista] - aparece uma solicitação aleatória recente de mod
+13:00 BanchoBot: ROLL <número> - rola um dado e obtenha um resultado aleatório de 1 para o número (padrão 100)
```
Comment (Walavouchey) [inline wiki/BanchoBot/pt-br.md:52]: don't translate this, as the comment below says. this shows what the `!help` command displays in the chat

File: wiki/BanchoBot/pt-br.md [hunk 2]
```diff
@@ -1,40 +1,160 @@
...
+
+```
+13:02 pippi: !where Ephemeral
+13:02 BanchoBot: Ephemeral is in Australia
+```
+
+### FAQ
+
+```
+!faq <entry>
```
Comment (Walavouchey) [inline wiki/BanchoBot/pt-br.md:119]: and here

File: wiki/BanchoBot/pt-br.md [hunk 3]
```diff
@@ -1,40 +1,160 @@
 ---
-outdated: true
+tags:
+  - bancho
+  - server bot
+  - commands
+  - announcements
```
Comment (Walavouchey) [inline wiki/BanchoBot/pt-br.md:6]: you can also add translations for these tags below them

**Comment by GranularBow8193** [comment]
I fixed all, any more problems?

File: wiki/BanchoBot/pt-br.md [hunk 4]
```diff
@@ -1,40 +1,160 @@
 ---
-outdated: true
+tags:
+  - bancho
+  - server bot
+  - comandos
+  - anúncios
```
Comment (Walavouchey) [inline wiki/BanchoBot/pt-br.md:6]: the original tags still need to be there

```suggestion
  - commands
  - announcements
  - comandos
  - anúncios
```

**Comment by Walavouchey** [comment]
no need to update the branch all the time, since it'll be done before merging anyway

**Comment by GranularBow8193** [comment]
ok, i'm still learning how to use the site LOL
btw how do i "mention" an existing path of the osu!wiki in the title? and have that little gray bkg in the "BanchoBot" (where you can click and get right in to the path)

**Comment by Walavouchey** [comment]
it's called a code block, you do it using backticks: \`BanchoBot`

**Comment by GranularBow8193** [comment]
oh, I see, thanks for the tip

**Comment by GranularBow8193** [comment]
@LeoFLT

**Review by Walavouchey — APPROVED** [review]
merging since it's been two weeks

**Pull request merged**